pipelines/whale/utils/extractor_wrappers.py

```diff
@@ -52,6 +55,25 @@ def add_ugc_runner(extractors: list, conf: ConfigTree, connection):
     return extractors, conf
 
 
+def add_indexes(extractors: list, conf: ConfigTree, connection):
```

Comment:
Ha. Thanks for following my pattern here - it is a bit weird though, isn't it? Probably would make sense to refactor at some point...

Reply:
Well, I am sure it could be improved, but I don't think it should block us from making progress. Let me see if I can come up with a refactor.

Reply:
Oh no didn't mean to block the review. Just got bogged down checking things before merging :)